### PR TITLE
Comments: when more than one is expanded, adding/editing is buggy #3243

### LINKED ISSUE #3243: Comments: when more than one is expanded, adding/editing is buggy
Pardon the reduced screen size, I wanted to make sure everything can fit in one screenshot.
commentbug3
Basically, I opened more than one session in the comments page. When I clicked the button for the second one...
commentbug4
Guess which one gets opened.
By the way, this problem also happened when I clicked the 'Add comment' button, but not if 'Delete comment'.

### PR DESCRIPTION
**Pull request by tangzijian**
Fixes #3243

### CODE REVIEW

**Comment by tangzijian** [comment]
Hi,
I haven' modified the test cases for this fix.
The problem of this issue is the feedback session index (fsIndx) in InstructorFeedbackResponseCommentsLoad.jsp is wrong. That causes every feedback session has an index of 1.
I fixed the problem by passing the correct feedback session index through a parameter of the URL. I used Integer.parseInt() to convert the string of the index back into integer. What is the best way to handle the exception? catch internally or just throw at method declaration?
Thanks.

File: src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java [hunk 1]
```diff
@@ -29,6 +29,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
         Assumption.assertNotNull(courseId);
         String fsname = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        int fsindex = Integer.parseInt(getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_INDEX));
```
Comment (kai33) [inline src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java]: Need to verify the param is not null;
then parse it with try-catch; if fail to parse, fast-fail it.

Comment (tangzijian) [inline src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java]: @kai33 what do you mean by fast-fail it?

Comment (kai33) [inline src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java]: Assumption.fail(some msg)

Comment (tangzijian) [inline src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java]: Thanks.

**Comment by tangzijian** [comment]
Hi, 
I have fixed this issue, please review.
Thanks.

**Comment by tangzijian** [comment]
Can someone review this please?
Thanks.

**Comment by kai33** [comment]
hi pls merge or rebase with the recent master branch

**Comment by tangzijian** [comment]
Done.

**Comment by kai33** [comment]
Thank you for your contribution : )

**Pull request merged**